client/src/pages/Map/MapLayers.js

```diff
@@ -126,16 +113,12 @@ export default function MapLayers({
               type: 'FeatureCollection',
             },
           },
-          ...circleLayerZoomRangeGeo,
+          ...circleLayerZoomRange,
           paint: {
             'circle-color': 'transparent',
-            ...circleRadius,
+            'circle-radius': linearZoom,
             'circle-stroke-color': 'white',
-            'circle-stroke-width': [
-              'interpolate', ['linear'], ['zoom'],
-              12, 2,
-              24, 12,
-            ],
+            'circle-stroke-width': linearZoom,
```

Comment:
Making the stroke width the same as the radius seems like it'll make the selection border way too heavy.